modules/apps/foundation/portal-configuration/portal-configuration-test-util-test/src/testIntegration/java/com/liferay/portal/configuration/test/util/test/ConfigurationTemporarySwapperTest.java

```diff
@@ -167,8 +167,6 @@ public void testWillPreservePreviouslySavedProperties() throws Exception {
 
 		testConfiguration.update(temporaryValues);
 
-		_persistenceManager.load(_pid);
-
```

Comment:
@drewbrokke I don't think that's the case, as far as I know, updating a Configuration object is happening in a synced way, which means the store action happens right away inside the PersistenceManager. It is the service who receives the Configuration update is happening in an async way, not the configuration update itself.